guides/common/modules/proc_remediating-compliance-failures.adoc

```diff
@@ -0,0 +1,33 @@
+[id="remediating-compliance-failures_{context}"]
+= Remediating compliance failures
+
+You can examine compliance reports and, in some cases, remediate cases of non-compliance.
+
+[WARNING]
+====
+Always test the recommended remedial actions or scripts in a non-production environment before implementing them in production.
+Remediation might render the system non-functional.
+====
+
+.Prerequisites
+* Your user account has a role assigned that has the `view_arf_reports` and `view_hosts` permissions.
+
+.Procedure
+To examine a simplified compliance policy report:
```

Comment:
```suggestion
.Examining a simplified compliance policy report
```

same line 22. counter argument: a procedure always contains exactly one ".Procedure" marker. Your call.

Reply:
Yeah, I considered this too. I'd prefer to keep it all under one `.Procedure` heading because both subprocedures are actually about remediating (or will be soon, see my other comment) and this helps keep them nicely together..? It's not a great explanation, but I'd say let's keep it the way it is now.